docs/source/specification/terms_and_model/variation.rst

```diff
@@ -61,8 +61,17 @@ interval (e.g., due to one or more indels).
 * The VMC definition of Allele applies to all Sequence types (DNA, RNA, AA).
 
 
-Text Variant
-@@@@@@@@@@@@
-.. todo::
+.. _text:
+
+Text Variation
+@@@@@@@@@@@@@@
+
+**Biological definition:** None.
+
+**Computational definition:** The Text class is intended to capture textual descriptions of variation that cannot be parsed by other Variation subclasses, but are still treated as variation.
+
+**Implementation guidance**
 
-   Finish Text Variant
+* An implementation MUST represent Variation with subclasses other than Text if possible.
+* An implementation SHOULD define or adopt conventions for defining the strings stored in Text.definition.
+* If a future version of VR-Spec is adopted by an implementation and the new version enables defining existing Text objects under a different Variation subclass, the implementation MUST construct a new object under the other Variation subclass. In such a case, an implementation SHOULD persist the original Text object and respond to queries matching the Text object with the new object.
```

Comment:
I think we're crossing over from schema to the behavior of a REST interface. (I agree with you, I just think it's out of scope.)

Reply:
Is this the right place to encourage implementers to contact us with proposals for new variation types? Up to you.

Reply:
Thanks @reece. I'm going to leave the guidance section recommendations barring strong objections. I like the idea of encouraging implementers to contact us; will add a **notes** section with this.